org.gnucash.GnuCash.json

```diff
@@ -0,0 +1,168 @@
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/project/libdbi/libdbi/libdbi-0.9.0/libdbi-0.9.0.tar.gz",
+          "sha256": "dafb6cdca524c628df832b6dd0bf8fabceb103248edb21762c02d3068fca4503"
+        }
+      ]
+    },
+    {
+      "name": "libdbi-drivers",
+      "config-opts": [
+        "--with-dbi-libdir=/app/lib/",
+        "--with-dbi-incdir=/app/include/",
+        "--with-sqlite3"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/project/libdbi-drivers/libdbi-drivers/libdbi-drivers-0.9.0/libdbi-drivers-0.9.0.tar.gz",
+          "sha256": "43d2eacd573a4faff296fa925dd97fbf2aedbf1ae35c6263478210c61004c854"
+        }
+      ]
+    },
+    {
+      "name": "libart-gpl",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/libart_lgpl/2.3/libart_lgpl-2.3.21.tar.bz2",
+          "sha256": "fdc11e74c10fc9ffe4188537e2b370c0abacca7d89021d4d303afdf7fd7476fa"
+        }
+      ]
+    },
+    {
+      "name": "libglade",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libglade/2.6/libglade-2.6.4.tar.bz2",
+          "sha256": "64361e7647839d36ed8336d992fd210d3e8139882269bed47dc4674980165dec"
+        }
+      ]
+    },
+    {
+      "name": "libgnomecanvas",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libgnomecanvas/2.30/libgnomecanvas-2.30.3.tar.bz2",
+          "sha256": "859b78e08489fce4d5c15c676fec1cd79782f115f516e8ad8bed6abcb8dedd40"
+        }
+      ]
+    },
+    {
+      "name": "libgsf",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.gnome.org/pub/gnome/sources/libgsf/1.14/libgsf-1.14.41.tar.xz",
+          "sha256": "150b98586a1021d5c49b3d4d065d0aa3e3674ae31db131af5372499d2d3f08d3"
+        }
+      ]
+    },
+    {
+      "name": "goffice",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/goffice/0.8/goffice-0.8.17.tar.xz",
+          "sha256": "165070beb67b84580afe80a8a100b674a81d553ab791acd72ac0c655f4fadb15"
+        },
+        {
+          "type": "patch",
+          "path": "goffice-0.8-use-pkgconfig-for-pcre.patch"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "rm -f configure"
+          ]
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoconf"
+          ],
+          "dest-filename": "autogen.sh"
+        }
+      ]
+    },
+    {
+      "name": "webkitgtk",
+      "config-opts": [
+        "--disable-webkit2",
+        "--with-gtk=2.0"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://webkitgtk.org/releases/webkitgtk-2.4.11.tar.xz",
```

Comment:
That's the exact point why I'm working on this – it's not going to stay in repos forever, and there is long way ahead before upstream moves to supported libraries.